src/hooks/screen/useProfileScreenHook.js

```diff
@@ -27,10 +28,9 @@ const useProfileScreenHook = () => {
 
   const fetchAnonymousPost = async (offset = 0, limit = 10) => {
     setIsLoadingFetchingAnonymousPosts(true);
-    const cacheFeed = StorageUtils.myAnonymousFeed.get();
     const response = await ProfileRepo.getSelfAnonymousFeed(offset, limit);
     const {data: myAnonymousFeed} = response;
-    const {mapNewData} = mappingColorFeed({dataFeed: myAnonymousFeed, dataCache: cacheFeed});
+    const mapNewData = mappingColorFeed(myAnonymousFeed);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The cache retrieval logic has been removed from the `fetchAnonymousPost` function. This could potentially impact performance if the data was previously being cached for faster access. If the removal of caching was intentional due to some other considerations (like data freshness), then this change is fine. Otherwise, consider adding back the cache retrieval logic.
```diff
- const cacheFeed = StorageUtils.myAnonymousFeed.get();
- const {mapNewData} = mappingColorFeed({dataFeed: myAnonymousFeed, dataCache: cacheFeed});
+ const mapNewData = mappingColorFeed(myAnonymousFeed);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
closed